src/server/rpc/rpc-manager.js

```diff
@@ -39,8 +39,14 @@ RPCManager.prototype.loadRPCs = function() {
         .map(name => [name, path.join(PROCEDURES_DIR, name, name+'.js')])
         .filter(pair => fs.existsSync(pair[1]))
         .map(pair => [pair[0], require(pair[1])])
-        .filter(pair => typeof pair[1] === 'function' ||
-            (!!pair[1] && !_.isEmpty(pair[1])))
+        .filter(pair => {
+            let service = pair[1];
+            if (typeof service === 'function' || !!service && !_.isEmpty(service)) {
+                if(typeof service.isSupported === 'function') return service.isSupported();
```

Comment:
this could be simplified by just checking `if (service.isSupported) return service.isSupported()`. We shouldn't have to check if it is a function because our dev API for creating RPCs dictates that it has to be a function. That is, if it isn't a function, then it is a bug and silently failing might actually be a bad thing - having the server crash would give the developer faster feedback about mistakes in the RPC.

Reply:
Also, could you print a console message saying that the given service is not going to be loaded?

Reply:
the check to see if service is a fn is already in there and I'm trying to make as little a change as possible in this PR